src/screens/StudentGroup.tsx

```diff
@@ -92,23 +91,73 @@ const StudentGroup: React.FC = () => {
 
   return (
     <>
-      <BodyBoxResponseHeight>
+      <Box
+        display="grid"
+        gridTemplateRows={{ base: '5rem max-content 1fr max-content max-content', md: '5rem 5rem .5fr 2fr 1fr' }}
+        gridTemplateColumns={{
+          base: '1fr minmax(150px, max-content) 1fr',
+          md: 'minmax(min-content, 50%) minmax(40%, 50%)',
+          lg: '10% 40% 40% 10%',
+          xl: '15% 35% 35% 15%',
+        }}
+        gridTemplateAreas={{
+          base: `'top top top'
+                '. name .'
+                '. pic .'
+                '. fact .'
+                'bottom bottom bottom'`,
+          md: `'top top'
+                '. .'
+                'pic name'
+                'pic fact'
+                'pic bottom'`,
+          lg: `'top top top top'
+          '. . . . '
+              '. pic name .'
+              '. pic fact .'
+              '. pic bottom .'`,
```

Comment:
try more descriptive names here i think, great idea though

Reply:
Thanks!  I find it really hard to name containers for multiple unrelated things.  One of the areas that I called "bottom" sits at the bottom of the screen but has two things in it: the student list and the next button.  Not sure how to stay concise enough for the grid-template-areas definition not to look god-awful but still be descriptive. I'll work on this and find a nice middle ground